emails/apps.py

```diff
@@ -19,7 +19,12 @@ def __init__(self, app_name, app_module):
             self.ses_client = boto3.client(
                 'ses', region_name=settings.AWS_REGION
             )
-            self.s3_client = boto3.client('s3', region_name=settings.AWS_REGION)
+            self.s3_client = boto3.client(
+                's3',
+                region_name=settings.AWS_REGION,
+                max_attempts=1,  # this includes the initial attempt to get the email
+                retry_mode='standard',
+            )
```

Comment:
I think we should model after the example code in [Defining a retry configuration in a Config object for your Boto3 client](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html#defining-a-retry-configuration-in-a-config-object-for-your-boto3-client):

```python
import boto3
from botocore.config import Config

config = Config(
   retries = {
      'max_attempts': 10,
      'mode': 'standard'
   }
)

ec2 = boto3.client('ec2', config=config)
```

That looks like this (without the ``from botocore.config import Config``):


```suggestion
            s3_config = Config(
                region_name=settings.AWS_REGION,
                retries = {
                    "max_attempts": 1,  # this includes the initial attempt to get the email
                    "mode": "standard",
                })
            self.s3_client = boto3.client('s3', config=s3_config)
```

Also see the [botocore.config.Config](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html) docs.